docs/pages/usage-billing.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Usage Reporting and Billing
+description: Provides a detaield breakdown of Teleport usage reporting and billing.
+---
+
+Teleport Team, Teleport Enterprise Cloud, and Teleport Enterprise send
+anonymized usage data to Teleport so we can calculate billing metrics. This
+guide describes the anonymized usage data and the billing metrics we calculate.
```

Comment:
I think this is ok as is. We want the default to be that it does. The customer can negotiate a clause in their sales order if they do not want it to report usage.

Reply:
@ptgott I would remove the specific plans because we could have plans in the future that also report usage but are not specified here. Something like this:

"Teleport receives anonymized usage data so we can calculate billing metrics. This guide describes the anonymized usage data and the billing metrics we calculate."